### ISSUE TITLE
Deprecation of older cipher suites for Cloudant

### ISSUE THREAD

**Issue by meyumeapps** [cloudant/swift-cloudant#193]
Do we know if the library will still work when IBM end support for a few outdated TLS v1.2 cipher suites. After this date, clients will not be able to connect to Cloudant using the following cipher suites: *ECDHE-RSA-AES128-SHA256* *ECDHE-RSA-AES256-SHA384* *AES128-GCM-SHA256* *AES256-GCM-SHA384*

```
Deprecation of older cipher suites for Cloudant

Type: Announcement
Component: Cloudant
Regions: Dallas, London, Chennai, Washington DC, Toronto, Frankfurt, Osaka, Seoul, Sao Paulo, Tokyo, eu-fr2, Sydney
Start Time: 16 Aug 2021, 8:06 PM UTC
Update Time: 17 Aug 2021, 3:10 PM UTC

On September 14, 2021 we will end support for a few outdated TLS v1.2 cipher suites. After this date, clients will not be able to connect to Cloudant using the following cipher suites: *ECDHE-RSA-AES128-SHA256* *ECDHE-RSA-AES256-SHA384* *AES128-GCM-SHA256* *AES256-GCM-SHA384* The transition should be completely transparent to virtually all clients but there is a small risk that an old client might not be able to connect to Cloudant. If you are unable to connect to the Cloudant service after this date please upgrade to a newer version of your client or cryptography library (e.g. openssl), or open a support ticket for assistance. 
```

**Comment by ricellis** [comment]
_TLDR;_ if you haven't changed any default TLS configuration it will still work

The TLS support is provided by the underlying [Swift `URLSession`](https://developer.apple.com/documentation/foundation/urlsession) which in turn is governed by [App Transport Security (ATS)](https://support.apple.com/en-gb/guide/security/sec100a75d12/web) (which is the default for apps made for iOS 9 or later and macOS 10.11 or later).

By default on modern iOS and macOS versions TLSv1.3 will be used by default anyway, if for some reason TLSv1.2 is being then ATS and Cloudant's prefered cipher order will negotiate to a more secure cipher than those listed.

For Swift on Linux I believe the ciphers are dependent on the underlying `openssl` version, but any recent version combined with Cloudant's preferred cipher list should already be negotiating to use something more secure than the ciphers listed for removal.

**Comment by meyumeapps** [comment]
Thanks @ricellis 👍
